### ISSUE TITLE
parameters: enable std

### ISSUE THREAD

**Issue by tbates** [tbates/umx#92]
```r
require(umx)
manifests = names(demoOneFactor)
m1 <- umxRAM("OneFactor", data = demoOneFactor,
	umxPath(latents, to = manifests),
	umxPath(v.m. = manifests),
	umxPath(v1m0 = latents)
)
parameters(m1, patt="one_to",std=TRUE)
```

> Error in parameters(m1, patt = "one_to", std = TRUE) : 
  Sorry, std not implemented yet: Standardize the model and provide this or the summary as input.

**Comment by tbates** [comment]
now works with all models which `umx_standardize()` knows how to std

```R
require(umx)
manifests = names(demoOneFactor)
m1 <- umxRAM("OneFactor", data = demoOneFactor,
	umxPath(latents, to = manifests),
	umxPath(v.m. = manifests),
	umxPath(v1m0 = latents)
)
parameters(m1, patt="one_to",std=TRUE)
```

**Comment by mcneale** [comment]
Very nice!!  Hope all’s well in your cell in Edinburgh.  No I am not 'Liz 1 :).

Cheers
Mike

Michael C. Neale, Ph.D.
Professor, Virginia Institute for Psychiatric and Behavioral GeneticsIPBG
Virginia Commonwealth University
Shipping Address: 800 East Leigh St. Suite 1-156, Richmond VA 23219-1534
Mailing Address : Box 980126 MCV Richmond VA 23298-0126
Telephone: (804) 828-3369  Email: michael.neale@vcuhealth.org<mailto:michael.neale@vcuhealth.org>

On Mar 18, 2020, at 3:14 PM, Tim Bates <notifications@github.com<mailto:notifications@github.com>> wrote:



Closed #92<https://urldefense.com/v3/__https://github.com/tbates/umx/issues/92__;!!JqxBPMk!x7xAGBvTMHNR8c6NbGvUjDdBgSBy09Z-JM-dBW0H2wC-_W98FU_P5AhspYwsvFwlNg$>.
